webrtc.html

```diff
@@ -7051,10 +7051,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <p>The <code>failed</code> and <code>completed</code> states require an
       indication that there are no additional remote candidates. This can be
-      indicated either by <a data-link-for=
+      indicated by calling <code><a>addIceCandidate</a></code> with
+      a candidate value whose <code>candidate</code> property is set
+      to an empty string, by <a data-link-for=
       "RTCPeerConnection">canTrickleIceCandidates</a> being set to
-      <code>false</code>, or the processing of an end-of-candidates indication
-      as described in [[!JSEP]].</p>
+      <code>false</code>, or the processing of an end-of-candidates
+      indication as described
+      in <span data-jsep="addicecandidate">[[!JSEP]]</span>.</p>
```

Comment:
So the truth is that I still don't believe JSEP anywhere actually describes processing of an end-of-candidates indication but do believe that I'm never going to get a proper answer on that, so I almost deleted the reference to JSEP altogether.  I am happy to delete everything from ', or the processing . . .' onwards.